cmd/network.go

```diff
@@ -123,52 +123,6 @@ func newNetworkCmd(c *config) *cobra.Command {
 		ValidArgsFunction: c.comp.NetworkListCompletion,
 	}
 
-	prefixCmd := &cobra.Command{
```

Comment:
When the user can do this now all through the update command, we can drop these commands. No need for two ways.